rocky/account/views/account.py

```diff
@@ -13,31 +15,36 @@ class PageActions(Enum):
     WITHDRAW_ACCEPTANCE = "withdraw_acceptance"
 
 
-class AccountView(OrganizationView, DetailView):
-    template_name = "account_detail.html"
-    context_object_name = "member"
-
-    def get_object(self):
-        return OrganizationMember.objects.get(user=self.request.user, organization=self.organization)
-
+class OOIClearanceManager(OrganizationView, TemplateView):
     def post(self, request, *args, **kwargs):
         if "action" in self.request.POST:
             self.handle_page_action(request.POST["action"])
         return self.get(request, *args, **kwargs)
 
     def handle_page_action(self, action: str):
         try:
-            organizationmember = self.get_object()
             if action == PageActions.ACCEPT_CLEARANCE.value:
-                organizationmember.acknowledged_clearance_level = organizationmember.trusted_clearance_level
+                self.organization_member.acknowledged_clearance_level = self.organization_member.trusted_clearance_level
             elif action == PageActions.WITHDRAW_ACCEPTANCE.value:
-                organizationmember.acknowledged_clearance_level = -1
+                self.organization_member.acknowledged_clearance_level = -1
             else:
                 raise Exception(f"Unhandled allowed action: {action}")
-            organizationmember.save()
+            self.organization_member.save()
         except RequestException as exception:
```

Comment:
I know that this is already there, but the catching of `RequestException` doesn't really make sense here because we don't do a request. I think we remove the try/except.